components/fms-consumer/src/main.rs

```diff
@@ -36,13 +36,60 @@ use rdkafka::consumer::Consumer;
 use rdkafka::message::{BorrowedHeaders, BorrowedMessage, Headers};
 use rdkafka::{ClientConfig, Message};
 
+
+use async_std::task::sleep;
+use futures::prelude::*;
+use futures::select;
+use zenoh::config::Config;
+use zenoh::prelude::r#async::*;
+
+
 const CONTENT_TYPE_PROTOBUF: &str = "application/vnd.google.protobuf";
 
 const HEADER_NAME_ORIG_ADDRESS: &str = "orig_address";
 
 const PARAM_KAFKA_PROPERTIES_FILE: &str = "kafka-properties-file";
 const PARAM_KAFKA_TOPIC_NAME: &str = "kafka-topic";
 
+const SUBCOMMAND_KAFKA: &str = "kafka";
+const SUBCOMMAND_ZENOH: &str = "zenoh";
+
+const KEY_EXPR: &str = "fms/vehicleStatus";
+
+pub fn parse_args(args: &ArgMatches) -> Config {
```

Comment:
You can specify this only for Zenoh, like naming it as parse_zenoh_args or something else.
Also while the function is only used in the file, please remove pub.

Reply:
update the code.